gekitchen/erd/serializer.py

```diff
@@ -0,0 +1,80 @@
+import logging
+from datetime import timedelta
+from textwrap import wrap
+from typing import Any, Dict, Optional, Set, Tuple, Union
+
+from .codes import ErdCode
+from .registry import _registry
+from .converters import ErdValueConverter
+from .converters.primitives import *
+
+_LOGGER = logging.getLogger(__name__)
+
+ErdCodeType = Union[ErdCode, str]
+
+class ErdSerializer:
+    def translate_erd_code(self, erd_code: ErdCodeType) -> ErdCodeType:
+        """
```

Comment:
```suggestion
    @staticmethod
    def translate_erd_code(erd_code: ErdCodeType) -> ErdCodeType:
        """
```

Reply:
We should talk static vs instance.  I'm coming from other languages (day job is C#) and generally try to avoid static stuff.  It's probably not the pythonic way though...